README.md

```diff
@@ -69,3 +69,36 @@ The crate has one optional feature: `static-openssl`.
 When the feature is enabled, `openssl` is linked statically against a locally compiled OpenSSL.
 This can be used to create a binary with a minimal set of runtime dependencies,
 and it can make compilation easier on systems with no recent version of OpenSSL readily available.
+
+On windows, this has been reported to work.
+```
+choco install openssl
+set OPENSSL_DIR=C:\Program Files\OpenSSL-Win64
+```
+
+## Examples
+multipart-stdin is an example of processing stdin and environment variables to support multipart upload.
+
+Build the example
+```
+ cargo run --example multipart-stdin --features static-openssl
+```
+
+Add the hook (windows example)
+```
+  - url: "/multipart-stdin"
+    commands:
+      - cmd: ["cmd.exe","/c","target\\debug\\examples\\multipart-stdin.exe"]
+        stdin: request-body
+    environment: ["OUTPUT_FOLDER=uploads","COUNT_SUFFIX","APPLY_TIMESTAMP"]
+```
```

Comment:
```suggestion
Add the hook:
```yaml
  - url: "/multipart-stdin"
    commands:
      - cmd: ["target/debug/examples/multipart-stdin"]
      # For Windows, add `.exe` and use escaped backslashes for the path:
      # cmd: ["target\\debug\\examples\\multipart-stdin.exe"]
        stdin: request-body
    environment: 
      OUTPUT_FOLDER: uploads
      COUNT_SUFFIX: 1
      APPLY_TIMESTAMP: 1
```

I don't mind documenting things for Windows, but I do want to keep Unix platforms the primary example. I also don't think you need cmd if you just want to run a binary.

Reply:
I commented the cert stuff out since it stops us from using it out of the box as no one has those files.

What about creating win32 endpoints /print-env-win32  for the pwd/env example.  Really, I was looking to add one example of using cmd.exe and it would be nice if that example were actually usable without modifications.

the same is true for the multipart-stdin.  part of me wants to test for windows and do a translation of / on the first command argument so that the same endpoint could work for both platforms without modification.  if we just create another endpoint with a win32 postfix both platforms have a place to go.

Reply:
Hmm, then how about adding an example-config.windows.yaml? I'd even be okay with renaming example-config.yaml to example-config.unix.yaml :p

Reply:
```
 22   - url: "/multipart-stdin"
 21     commands:
 20       - cmd: ["target/debug/examples/multipart-stdin"]
 19         stdin: request-body
 18     environment:
 17       OUTPUT_FOLDER: uploads
 16       COUNT_SUFFIX:
 15 #      APPLY_TIMESTAMP: 1
```

I just tested turning the env vars off and on.  The "target/debug/examples/multipart-stdin" value works as expected and no additional changes are required, there's one way to do it.

Reply:
Sounds good to me :)

Reply:
I don't care much one way or other about the cmd.exe endpoints.  Let me know how you'd like to proceed.

Reply:
Ok, I think we're close.  The code is there.  There's gotta be another change still right? :)

That indexmap cleaned things up nicely.